Description: Packaging for ArcheOS
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 img2dcm (0.1.0-1) unstable; urgency=low
 .
   * source package automatically created by stdeb 0.6.0+git
Author: Romain Janvier <romain.janvier@hotmail.fr>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ img2dcm-0.1.0/Readme.md
@@ -0,0 +1,2 @@
+This repository is a fork of https://bitbucket.org/tfmoraes/img2dcm.
+It contains some littles fixes in order to package img2dcm for Archeos 5.0.
--- /dev/null
+++ img2dcm-0.1.0/README.md
@@ -0,0 +1,2 @@
+img2dcm
+=======
\ No newline at end of file
--- /dev/null
+++ img2dcm-0.1.0/MANIFEST
@@ -0,0 +1,4 @@
+# file GENERATED by distutils, do NOT edit
+img2dcm
+ivDicom.py
+setup.py
